### ISSUE TITLE
[Medium/Difficult] Integrate PlayOnLinux v5 to canonical Rosetta

### ISSUE THREAD

**Issue by qparis** [PlayOnLinux/POL-POM-5#4]
Currently, POLv4 strings automatically sent to Rosetta for translation. Once translated, they are fetched back into the project

We need to have the same mechanism for POLv5 

For more details, see: (Separated PlayOnLinux_Tools github repository) 
- rebuildTemplate.sh (generate .po templates from source tree)
- update_template_from_launchpad (Get back translations from launchpad)

**Comment by yawkat** [comment]
Sounds difficult to do, maybe make a maven plugin that uses asm to analyze class files for this?

**Comment by qparis** [comment]
I think we just need to modify the current scripts. It is really a separate task from the project :)

**Comment by plata** [comment]
I used Rosetta quite a bit in the past and I have to say that I really hate it by now. It's often slow and buggy which is really frustrating for people doing translations in their spare time. Therefore, I would highly appreciate if we could use something else.

I have used Transifex and find it quite nice. Pricing could be an issue.

Maybe Weblate or Pootle could be an alternative. We could probably integrate those into the Phoenicis website so it's really easy for users to help with the translation.
